service/src/main/java/org/broadinstitute/listener/config/AppConfiguration.java

```diff
@@ -30,6 +35,17 @@ public TargetResolver targetResolver() {
     return new DefaultTargetResolver(properties);
   }
 
+  @Bean
+  public SamResourceClient samResourceClient() {
```

Comment:
Is `SamResourceClient` thread safe? I think this bean will have singleton scope by default; if so it will be shared across all threads that inject it.

Reply:
Idk sprint boot well... any comments @giventocode 

will merge this PR for now and address retrospectively if update is needed